docs/uns-use-the-network/cli.md

```diff
@@ -459,28 +468,20 @@ Set (add or update) properties of UNIK token.
 #### Parameters
 
 - `--unikid` (required): the ID of the UNIK token
-- `-p --properties` (required): Array of properties to set.
-  
-  `"key1:value1" "key3:"` Sets `value1` to `key1` and empty string to `key3`. See [allowed property key format](/uns-use-the-network/cheatsheet.html#property-keys-of-unik)
-- `--await` : Number of blocks to wait to get confirmed for the success. Default to `3`.
-  
-  `0` for immediate return.
-  Needs to be strictly greater than `--confirmation` flag
-- `--confirmations` : Number of confirmations to wait to get confirmed for the success. Default to `1`.
-
-  Needs to be strictly lower than `--await` flag
+- `-p --properties` (required): List of key/value to set as UNIK properties: "key1:value1" "key2:value2".  See [allowed property key format](/uns-use-the-network/cheatsheet.html#property-keys-of-unik)
+- `--await-confirmation` : Maximum number of blocks to wait to get one confirmation of the transaction. Default to 3. 0 for immediate return.
```

Comment:
is it really useful to repeat flag description ? 
Maybe yes.

Reply:
Yes its repetitive. But it is the same for all flags...